### PR TITLE
Set jsunity output to gtest style

### PR DESCRIPTION
**Pull request by KVS85**
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/5579/

### CODE REVIEW

File: js/common/modules/jsunity.js [hunk 1]
```diff
@@ -67,8 +68,8 @@ jsUnity.results.pass = function (index, testName) {
   RESULTS[testName].status = true;
   RESULTS[testName].duration = newtime - STARTTEST;
 
-  print(newtime.toISOString() + internal.COLORS.COLOR_GREEN +  ' [PASSED] ' + testName + internal.COLORS.COLOR_RESET +
-    ' in ' + STARTTEST.toISOString() + '+' + ((newtime - STARTTEST) / 1000).toFixed(3) + 's');
+  print(newtime.toISOString() + internal.COLORS.COLOR_GREEN +  ' [     PASSED ] ' + testName + internal.COLORS.COLOR_RESET +
+    ' (' + RESULTS[testName].duration + ' ms)');
```
Comment (dothebart) [inline js/common/modules/jsunity.js]: should we have setup/teardown here too?

Comment (KVS85) [reply js/common/modules/jsunity.js]: > I dislike not having the start time anymore - its what gives you the anchor to search for in server logs.

`jsUnity.results.beginSetUp` of `js/common/modules/jsunity.js` has `jsUnity.env.getDate().toISOString()` which indicates a timestamp for a start.

> should we have setup/teardown aded to this time here too?

`STARTTEST` is set at `jsUnity.results.begin` and used in `jsUnity.results.fail` and `jsUnity.results.pass` of `js/common/modules/jsunity.js` so we have it included.

**Comment by KVS85** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/5589/

**Comment by KVS85** [comment]
http://jenkins01.arangodb.biz:8080/job/arangodb-matrix-pr-linux/7244/

**Pull request merged**